src/importexport/imagesexport/internal/pdfwriter.cpp

```diff
@@ -65,9 +65,15 @@ Ret PdfWriter::write(INotationPtr notation, io::IODevice& destinationDevice, con
         return false;
     }
 
+    //transparency handling
+    const bool TRANSPARENT_BACKGROUND = options.find(OptionKey::TRANSPARENT_BACKGROUND) != options.end()
+        ? options.at(OptionKey::TRANSPARENT_BACKGROUND).toBool()
+        : configuration()->exportPdfWithTransparentBackground();
```

Comment:
`PngWriter` uses `muse::value` for this; might be good to use that here too, for consistency.

Reply:
can you clarify what we should do here?

Reply:
Basically, take a look at `PngWriter` (src/importexport/imagesexport/internal/pngwriter.cpp) to see how this is done there, namely like this:
```c++
    const bool TRANSPARENT_BACKGROUND = muse::value(options, OptionKey::TRANSPARENT_BACKGROUND,
                                                    Val(configuration()->exportPngWithTransparentBackground())).toBool();
```
And then do the same thing in PdfWriter.